src/main/java/Main.java

```diff
@@ -0,0 +1,46 @@
+import dao.ActionDao;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.Operation;
+import service.FruitService;
+import service.ParserService;
+import service.ReaderService;
+import service.WriterService;
+import service.impl.FruitServiceImpl;
+import service.impl.ParserServiceImpl;
+import service.impl.ReaderServiceImpl;
+import service.impl.WriterServiceImpl;
+import strategy.BalanceHandlerImpl;
+import strategy.OperationHandler;
+import strategy.OperationStrategy;
+import strategy.OperationStrategyImpl;
+import strategy.PurchaseHandlerImpl;
+import strategy.ReturnHandlerImpl;
+import strategy.SupplyHandlerImpl;
+
+public class Main {
+    private static final String FILE_INPUT = "src/main/resources/input.cvs";
+    private static final String FILE_OUTPUT = "src/main/resources/output.cvs";
+    private static final ReaderService readerService = new ReaderServiceImpl();
+    private static final ParserService parserService = new ParserServiceImpl();
+    private static final WriterService writerService = new WriterServiceImpl();
+    private static final Map<Operation, OperationHandler> operationHandlerMap = new HashMap<>();
+
+    public static void main(String[] args) {
+        List<String> dataReader = readerService.readFromFile(FILE_INPUT);
```

Comment:
dataReader name looks like some service name but this is just content of file represented as List<String>
```suggestion
        List<String> fileContent = readerService.readFromFile(FILE_INPUT);
```